app/messages.py

```diff
@@ -339,16 +335,26 @@
 LIST_TASK_COMMENTS_WITH_SUCCESS = {
     "message": "List task comments from a mentorship relation with success."
 }
-ADDITIONAL_INFO_SUCCESSFULLY_CREATED = {"message": "User additional info successfully created."}
-PERSONAL_BACKGROUND_SUCCESSFULLY_CREATED = {"message": "User personal background information successfully created."}
-ORGANIZATION_SUCCESSFULLY_CREATED = {"message": "Organization was created successfully."}
-ORGANIZATION_SUCCESSFULLY_UPDATED = {"message": "Organization was updated successfully."}
-PROGRAM_SUCCESSFULLY_CREATED = {
-    "message": "Program was created successfully."
+ADDITIONAL_INFO_SUCCESSFULLY_CREATED = {
+    "message": "User additional info successfully created."
+}
+PERSONAL_BACKGROUND_SUCCESSFULLY_CREATED = {
+    "message": "User personal background information successfully created."
+}
+ORGANIZATION_SUCCESSFULLY_CREATED = {
+    "message": "Organization was created successfully."
 }
-PROGRAM_SUCCESSFULLY_UPDATED = {
-    "message": "Program was updated successfully."
+ORGANIZATION_SUCCESSFULLY_UPDATED = {
+    "message": "Organization was updated successfully."
 }
+PROGRAM_SUCCESSFULLY_CREATED = {"message": "Program was created successfully."}
+PROGRAM_SUCCESSFULLY_UPDATED = {"message": "Program was updated successfully."}
+
+PROGRAM_NAME_IS_MISSING = {"message": "Program name was not provided."}
+
+
+PROGRAM_NAME_ALREADY_USED = {"message": "Program with that name already exists."}
```

Comment:
Why do these not start with new lines simialr to the change above?

Reply:
@ApheleiaS some messages are written from new lines because they exceed the length recommended while formatting using a code formatter(black in this case)